openslides_backend/presenter/get_mediafile_id.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Dict
+
+from .base import BasePresenter
+from .presenter import register_presenter
+from ..models.mediafile import Mediafile
+from ..shared.filters import FilterOperator
+import fastjsonschema
+from ..shared.schema import schema_version
+from ..shared.exceptions import PresenterException
+
+
+get_mediafile_id_schema = fastjsonschema.compile(
+    {
+        "$schema": schema_version,
+        "type": "object",
+        "properties": {
+            "meeting_id": {
+                "type": "integer",
+                "minimum": 1,
+            },
+            "path": {"type": "string"},
+        },
+        "required": ["meeting_id", "path"],
+        "additionalProperties": False,
+    }
+)
+
+
+@register_presenter("get_mediafile_id")
+class GetMediafileId(BasePresenter):
+    """
+    Retrieve an Id for the given mediafiel or None if it not exists or access is forbidden.
+    """
+    
+    schema = get_mediafile_id_schema
+
+    def get_result(self) -> Any:
+        # TODO: filter by meeting id
+        filter = FilterOperator("path", self.data["path"], "=")
```

Comment:
Operator is `==` not `=` I think.

Reply:
It's `=`: https://github.com/OpenSlides/OpenSlides/blob/8c57d911a30f22b41bbfe3b2b494bd805ffc6af4/docs/interfaces/datastore-service.txt#L266

Reply:
(Btw, it's this way because we inject it directly into the query and SQL also uses `=`)

Reply:
Ok then I have some bugs in other places of the code. See #79.